enclave/core/sgx/exit.S

```diff
@@ -51,11 +51,26 @@
 oe_asm_exit:
 .cfi_startproc
 
+.get_td:
+
+    // Save argument registers
+    push %rdi
+    push %rsi
+
+    // Get a pointer to the td structure. This value is expected to be present
+    // in %r9 for the remainder of oe_asm_exit
+    call oe_sgx_get_td
```

Comment:
instead of this call we could pass td as a parameter to oe_asm_exit

Reply:
Done